crates/net/network/src/network.rs

```diff
@@ -63,6 +64,7 @@ impl NetworkHandle {
             initial_sync_done: Arc::new(AtomicBool::new(false)),
             chain_id,
             tx_gossip_disabled,
+            tx_handle: Mutex::new(None),
```

Comment:
we don't need this, instead we should add another variant to message type that's send to the TransactionManager that gets a oneshot::Sender to send the handle